_posts/is-it-complex-break-it-down.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+header_image: /img/posts/is-it-complex-break-it-down/header.jpg
+title: "Is it complex? Break it down!"
+slug: is-it-complex-break-it-down
+author: Mauro Servienti
+synopsis: "Sometimes, we choose technology based on the perceived complexity or heaviness. We focus our decisions on the technical solutions rather than looking deeper at the problems we hire those solutions. Are we making the right choices?"
```

Comment:
> Sometimes, we choose technology based on the perceived complexity or heaviness. We focus our decisions on the technical solutions rather than looking deeper at the problems we hire those solutions

"hire" is the wrong word here but I'm not sure what the intent is so I can't think of an alternative. Maybe:

> We focus our decisions on the technical solutions and rather than looking deeper at the problems, we use those solutions

--or--

> We focus our decisions on the technical solutions and rather than looking deeper at the problems, we stick with what we know.

Reply:
```suggestion
synopsis: "Sometimes, we choose technology based on the perceived complexity or heaviness. We focus our decisions on the technical solutions and rather than looking deeper at the problems, we stick with what we know. Are we making the right choices?"
```